core/tests/test_utils.py

```diff
@@ -3092,11 +2859,10 @@ def __init__(self, f, exception, num_tries_before_success):
         super(FailingFunction, self).__init__(f)
         self._exception = exception
         self._num_tries_before_success = num_tries_before_success
-        self._always_fail = (
-            self._num_tries_before_success == FailingFunction.INFINITY)
+        self._always_fail = num_tries_before_success == self.INFINITY
```

Comment:
Perhaps put parens after the '=' -- not required, I know, but a bit easier to parse.